StreamVideoTests/Call/Call_Tests.swift

```diff
@@ -5,7 +5,6 @@
 @testable import StreamVideo
 import XCTest
 
-@MainActor
```

Comment:
Wasn't that sufficient?

Reply:
nope, since the base class is not a main actor.